communication/src/main/java/org/jboss/da/communication/aprox/impl/AproxConnectorImpl.java

```diff
@@ -82,9 +84,10 @@
             GraphExport export = mod.graph(req);
 
             if (export == null || export.getRelationships() == null) {
-                return Optional.empty();
+                // no dependencies found
+                return new GAVDependencyTree(gav);
```

Comment:
I was thinking if it is the best approach, because the analysis will fail. What about to return an empty tree and log the warning, that export is null?